app/core/config.py

```diff
@@ -0,0 +1,108 @@
+"""
+Environments and configuration file
+"""
+import logging
+import os
+import secrets
+from enum import Enum
+from functools import lru_cache
+from typing import Optional
+from pydantic import BaseSettings
+
+logger = logging.getLogger(__name__)
+
+
+class EnvironmentEnum(str, Enum):
+    """
+    Enumerate the application environments
+    """
+
+    PRODUCTION = "production"
+    LOCAL = "local"
+
+
+class GlobalConfig(BaseSettings):
+    """
+    Base configuration to be modified by specific environments
+    """
+
+    TITLE: str = "Open Brewery DB REST API"
+    DESCRIPTION: str = "A REST API for Open Brewery DB"
+    JWT_SECRET: str = secrets.token_urlsafe(32)
+    ACCESS_TOKEN_EXPIRE_MINUTES: int = 60 * 24 * 8
+    ALGORITHM = "HS256"
+    ENVIRONMENT: EnvironmentEnum
+    DEBUG: bool = False
+    TESTING: bool = False
+    TIMEZONE: str = "CDT"
+    # DB_USER: str = os.environ["DB_USER"]
+    # DB_PWD: str = os.environ["DB_PWD"]
+    # DB_HOST: str = os.environ["DB_HOST"]
+    # DB_NAME: str = os.environ["DB_NAME"]
+    # DATABASE_URL: Optional[
+    #     PostgresDsn
+    # ] = f"postgresql://{DB_USER}:{DB_PWD}@{DB_HOST}:5432/{DB_NAME}"
+    # DB_ECHO_LOG: bool = False
+    # Api V1 prefix
+    API_V1_STR = "/api/v1"
```

Comment:
Can we shorten this to `/v1`? `/api/` feels a little redundant.

Reply:
If you don't mind, this is one I would like to keep. That seems to be one of the more popular methods for versioning the REST API itself.

Reply:
I get it, but I'm envisioning:

`https://api.openbrewerydb.org/v1/breweries`

As opposed to:

`https://api.openbrewerydb.org/api/v1/breweries`

If it's _just_ the popular way, I'd prefer the OBDB example to be an alternative to show devs different ways to do things to get the same effect, especially when considering production environments, separation of systems, and subdomains.

Reply:
Thank you @milsman2 !